README.md

```diff
@@ -13,6 +13,10 @@ Here you will find a curated list of resources on a variety of topics including
 - [Python - Getting Started](PYTHON.md) †
 - [Angular](https://www.codecademy.com/resources/docs/general/angular) † and [Tour of Heroes application and tutorial](https://angular.io/tutorial) †
 - [Learn C#](https://www.codecademy.com/learn/learn-c-sharp) † and [Tutorial: Create a web API with ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api) †
+- [Getting Started with EF Core](https://learn.microsoft.com/en-us/ef/core/get-started/overview/first-app) † or [Entity Framework Core](https://www.entityframeworktutorial.net/efcore/entity-framework-core.aspx) †
+- [Tutorial: Get Started with Entity Framework 6 Code First using MVC 5](https://learn.microsoft.com/en-us/aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-an-entity-framework-data-model-for-an-asp-net-mvc-application) †
```

Comment:
I wonder if this isn't a better tutorial for us to use since it teaches a WebApi project instead of an MVC app.  https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-7.0&tabs=visual-studio

Reply:
Line 15 includes the link Matt noted: "Create a web API with ASP.NET Core" https://learn.microsoft.....
Therefore, do we keep line 16 and remove line 17 (which Matt notes might not be the best tutorial to use)?